src/neuroconv/basedatainterface.py

```diff
@@ -46,6 +49,30 @@ def get_metadata(self):
 
         return metadata
 
+    def synchronize(self, starting_time: Optional[float] = None, timestamps: Optional[ArrayType] = None):
+        """
+        Synchronize all time references for this interface to the common time basis.
+
+        This can be either an offset shift to the 'starting_time' of a regularly sampled series,
+        or a vector of 'timestamps'.
+
+        Both are set in units seconds relative to the common 'session_start_time'.
+
+        Parameters
+        ----------
+        starting_time: float
+            The starting time for all temporal data in this interface.
+        timestamps: ArrayType
+            A full vector of timestamps all temporal data in this interface.
+        """
+        if starting_time is not None or timestamps is not None:
+            assert starting_time != timestamps, "Specify either 'starting_time' or 'timestamps', but not both!"
```

Comment:
why the assert statement? Why not just throw an error? Are you permitting `timestamps == starting_time` as valid? Why?